Peltzman_effect_box/src/Peltzman.ino

```diff
@@ -0,0 +1,441 @@
+#include <FastLED.h>
```

Comment:
In this directory there are files `pseudocertainty_main.ino` and `shift_rand.ino`. Are they related or should they be deleted ? 

If there is only 1 source file per box how about naming the file `main.ino` instead ?

Reply:
pseudocertainty_main.ino is the old & incomplete version, i'll remove it. as for shift_rand.ino, i'm not exactly sure what it's meant for, but it's not used elsewhere. i will rename the proper .ino files into "main.ino" if necessary, but the arduino IDE will make a fuss about it if you try opening them (it will request a separate "main" folder to put "main.ino" inside, otherwise it won't open it).

Reply:
OK then. Please remove the old files and then use names which the Arduino IDE will like or adjust the dir structure to what the IDE needs (I don't know how that IDE actually works). Just keep it consistent for all directories.

Reply:
i've renamed the files to src.ino (so they can fit into the src folders without issues). also, i've removed the old files and i've adjusted the directory for the Salience bias box.